python-sdk/nuscenes/eval/predict/README.md

```diff
@@ -0,0 +1,73 @@
+# nuScenes predicton task
+In this document we present the rules, submission process, and evaluation metrics of the nuScenes prediction task.
+
+## Overview
+- [Introduction](#introduction)
+- [Participation](#participation)
+- [Challenges](#challenges)
+- [Submission rules](#submission-rules)
+- [Results format](#results-format)
+- [Evaluation metrics](#evaluation-metrics)
+
+## Introduction
+The goal of the nuScenes prediction task is to predict the future trajectories of objects in the nuScenes dataset.
+A trajectory is a sequence of x-y locations. For this challenge, the predictions are 6-seconds long and sampled at
+2 hertz.
+
+## Challenges
+To allow users to benchmark the performance of their method against the community, we will host a single leaderboard all-year round.
+Additionally, we organize a number of challenges at leading Computer Vision and Machine Learning conference workshops.
+Users that submit their results during the challenge period are eligible for awards.
```

Comment:
Be clear that awards may be different for each challenge.

Reply:
Added a sentence mentioning that!